app/models/admin_user.rb

```diff
@@ -26,5 +26,5 @@ class AdminUser < ApplicationRecord
   # Include default devise modules. Others available are:
   # :confirmable, :lockable, :timeoutable and :omniauthable
   devise :database_authenticatable,
-         :recoverable, :rememberable, :trackable, :validatable
```

Comment:
We think we can keep it for activeadmin, because in this case we are using cookies.